reliability/setup.yml

```diff
@@ -0,0 +1,179 @@
+---
+- name: Read config and create groups
+  hosts: localhost
+  vars:
+    cloudwatch_mon: false
+    pbench_mon: false
+  tasks:
+    - name: Include config file
+      include_vars:
+        file: ./config/config.yaml
+        name: env
+    - name: Add masters, nodes, etcds to groups
+      add_host:
+        name: "{{ item }}"
+        groups: masters
+      with_items: "{{ env.environment.masters }}"
+    - name: Add nodes to groups
+      add_host:
+        name: "{{ item }}"
+        groups: nodes
+      with_items: "{{ env.environment.nodes }}"
+    - name: Add etcds to groups
+      add_host:
+        name: "{{ item }}"
+        groups: etcds
+      with_items: "{{ env.environment.etcds }}"
+    - name: Set global variables
+      set_fact:
+        auth_type: "{{ env.environment.authtype }}"
+    - name: Read if cloudwatch is enabled
+      set_fact:
+        cloudwatch_mon: true
+      when: item == "cloudwatch"
+      with_items: "{{ env.environment.monitoring }}"
+    - name: Read if pbench is enabled
+      set_fact:
+        pbench_mon: true
+      when: item == "pbench"
+      with_items: "{{ env.environment.monitoring }}"
+
+- name: Configure reliability client/cleanup
+  hosts: localhost
+  tasks:
+    - name: Delete logs directory
+      file:
+        state: absent
+        path: logs
+    - name: Delete runtime directory
+      file:
+        state: absent
+        path: runtime
+    - name: Delete users.data
+      file:
+        state: absent
+        path: config/users.data
+    - name: Delete create users.data
+      file:
+        state: touch
+        path: config/users.data
+    - name: Create logs directory
+      file:
+        state: directory
+        path: logs
+    - name: Create runtime/keys directory
+      file:
+        state: directory
+        path: runtime/keys
+    - name: Create runtime/repos directory
+      file:
+        state: directory
+        path: runtime/repos
+    - name: Create runtime/templates directory
+      file:
+        state: directory
+        path: runtime/templates
+    - name: Install Ruby on client
+      yum:
+        name: ruby
+        state: present
+    - name: gem install packages
+      gem:
+        name: filewatcher, net-scp, net-ssh
+        state: present
+
+- name: Copy configs from master
+  hosts: masters
+  tasks:
+    - fetch:
+        src: /etc/origin/master/admin.kubeconfig
+        dest: runtime/keys
+        flat: yes
+    - fetch:
+        src: /etc/origin/master/ca.crt
+        dest: runtime/keys
+        flat: yes
+    - synchronize:
+        src: /usr/share/openshift/examples/quickstart-templates/*.*
```

Comment:
For containerized OCP installs the quickstart-templates are not available in this directory.  Instead we need to copy them from this directory on master:
/etc/origin/examples/quickstart-templates/*.*